### PR TITLE
Allow fetching weekly stats/points for players in a league context

### PR DESCRIPTION
**Pull request by kmudrick**

### CODE REVIEW

File: helpers/playerHelper.js [hunk 1]
```diff
@@ -15,21 +15,21 @@ exports.mapPlayer = function(player) {
         });
       });
     }
-
```
Comment (whatadewitt) [inline helpers/playerHelper.js]: I am actually so upset that these were in here and I didn't notice...

File: helpers/playerHelper.js [hunk 2]
```diff
@@ -78,6 +87,7 @@ exports.parseCollection = function(players, subresources) {
       switch (resource) {
         case 'stats':
           player.stats = self.mapStats(p[idx + 1].player_stats);
+          player.points = self.mapPoints(p[idx + 1].player_points);
```
Comment (whatadewitt) [inline helpers/playerHelper.js]: what sport is this for specifically? does it mess up any of the others?

Comment (kmudrick) [inline helpers/playerHelper.js]: Specifically for football. I don't play any other fantasy sports so I'm not sure - and I could not get the existing tests to run

Comment (whatadewitt) [inline helpers/playerHelper.js]: yea... the tests... they're kind of an ongoing project...

I've gotten more into doing them in my real life job, but adding them late now is not the most exciting thing with the limited free time I have (terrible, right?)

I'll run through some stuff on this branch tonight and get back to you!

Comment (kmudrick) [inline helpers/playerHelper.js]: This project is great - thank you - no need to apologize for not having free time :)

Personally, I might have some free time between jobs in a little bit, so I _might_ be able to help with tests if you think they'd be valuable.

**Comment by kmudrick** [comment]
If you are curious on the use case: I'm in a (football) league where we do additional prizes based on the team with the best offense and the team with the best defense - so I am literally looping through every week of the season (since American football operates on a 'weekly' schedule), for every team in this league, and getting the individual points for each player that started at an offensive position ('WR', 'W/R/T', 'TE', 'RB', 'QB', 'K') and the points for every player/team that started at a defensive position ('CB', 'DE', 'S', 'LB', 'DT', 'DEF', 'D') to compute that.

**Comment by whatadewitt** [comment]
... I'm going over this more and more and it almost looks like the exact solution that I was looking for for the "league.players" resource... do you mind if I take some small liberties with the code...

**Comment by whatadewitt** [comment]
Any chance I could ask you for your league ID and/or whether you can make it public so I can test the calls versus it? I only have MLB/NHL leagues that I can test against... and I want to see what I can actually get back as MLB has me pretty limited...

**Comment by whatadewitt** [comment]
I've been looking at this, and while it is causing a couple of smaller issues, I can easily work around that... what I'm more curious about is how you found this? I don't see it anywhere in the documentation that you can "chain" like this...

**Comment by whatadewitt** [comment]
... so... there was a bunch of stuff I didn't understand when you submitted this, and for that I apologize. I just pushed 3.2.0 which has this functionality in it. I wish I had taken some more time to go over this when you submitted it and wanted to thank you, immensely, for your time doing it!!

**Pull request closed without merging**